Move SAMBA_CHECK_UNDEFINED_SYMBOL_FLAGS to the end of configure().
If not, any compiler checks done afterwards will for some reason
hang the configure on ppc32 forever.

--- a/wscript
+++ b/wscript
@@ -452,8 +452,6 @@ def configure(conf):
     conf.RECURSE('lib/mscat')
     conf.RECURSE('packaging')
     conf.RECURSE('lib/krb5_wrap')

-    conf.SAMBA_CHECK_UNDEFINED_SYMBOL_FLAGS()
-
     # gentoo always adds this. We want our normal build to be as
     # strict as the strictest OS we support, so adding this here
     # allows us to find problems on our development hosts faster.
@@ -534,6 +532,8 @@ def configure(conf):
             conf.env.undefined_ldflags = []

     conf.SAMBA_CONFIG_H('include/config.h')
+
+    conf.SAMBA_CHECK_UNDEFINED_SYMBOL_FLAGS()

 def etags(ctx):
